rmf_traffic_ros2/src/rmf_traffic_ros2/schedule/Node.cpp

```diff
@@ -120,6 +161,18 @@ ScheduleNode::ScheduleNode(const rclcpp::NodeOptions& options)
 
   // TODO(MXG): As soon as possible, all of these services should be made
```

Comment:
We can probably delete this comment since I don't think we'll ever make the schedule node services multi-threaded, because the complexity wouldn't be worth any performance improvements.

Reply:
Removed in 1b7a0db.